### PR TITLE
Feat: dark mode

### PR DESCRIPTION
**Pull request by victormendoza96**
- create dark mode
- responsive for desktop (1060px)
tbkdarkmode

### CODE REVIEW

File: public/images/son.svg
Comment (mastudillot) [inline public/images/son.svg:1]: this should be named as sun

File: resources/views/webpay/commit.blade.php [hunk 1]
```diff
@@ -11,8 +11,10 @@
 
     <h2>Paso 1 - Datos recibidos:</h2>
     <ul class="mb-32">
-        <div class="m-32">Después de completar el flujo en el formulario de pago, recibirás un POST con la siguiente
-            información:</div>
+        <p class="m-32">
+            Después de completar el flujo en el formulario de pago, recibirás un POST con la siguiente
+            información:
+        </p>
```
Comment (mastudillot) [inline resources/views/webpay/commit.blade.php:17]: The method is GET

**Review by mastudillot — COMMENTED** [review]
check my comment

**Review by mastudillot — APPROVED** [review]
LGTM

**Review by gasspper — APPROVED** [review]
lgtm

**Pull request merged**